pkg/reconciler/cluster.go

```diff
@@ -214,12 +208,15 @@ func handleStatusGen(
 			cr.Status.State = string(clusterCreating)
 			return false
 		}
+
+		//		if lastKnown.UID != "" {
```

Comment:
I guess you will need to remind me about what the current state of affairs is for this PR. I think the outstanding issue was that we would always get the below "unknown" message for newly created clusters now, is that right?

If that's so, I'm slightly puzzled that `incoming` is non-empty in that case. Sooo maybe I'm misremembering.